### PR TITLE
Project Requirements

### PR DESCRIPTION
**Pull request by pountzas**
- You should follow the layout of the provided design. You should change only the titles, descriptions, and photos - in order to create a website about something other than motorcycles.
- Select a theme for your website - is it going to be a website for booking doctor appointments, booking online classes, or something else?

- The user logs in to the website, only by typing the username (a proper authenticated login is a requirement if your group is made of 5 people).

- In the navigation panel, the user can see links to:

 - Motorcycles/doctors/classes/items that you selected as a theme.
 - "Reserve" form.
 - "My reservations".
 - "Add motorcycle/doctor/class/item that you selected as a theme" (if you are in a group of 4 people the link is visible to everybody).
 - "Delete motorcycle/doctor/class/item that you selected as a theme" (if you are in a group of 4 people the link is visible to everybody).
 - On the main page, the user can see a list of motorcycles/doctors/classes/items that you selected as a theme.
 - When the user selects a specific item, they can see the details page with its full description (skip the "Rotate image" button).

- In the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).
- When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.

- Make the app responsive, creating both mobile and desktop versions.

### CODE REVIEW

File: src/components/MyAppointments.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const MyAppointments = () => (
+  <div className="responsive-flex">
+    <Nav />
+    <div>
+      <h1>My Appointments</h1>
+    </div>
+  </div>
+);
+
```
Comment (SpaYco) [inline src/components/MyAppointments.js:12]: My Appointments currently don't give any data, please fix that!

File: src/Pages/CarPage.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <h1 className="fw-bold">{car.name}</h1>
+        <h2>{car.model}</h2>
+        <p className="ms-5">{car.description}</p>
+        <Link
+          to="/appointments"
+          className="btn btn-secondary"
+          type="button"
+        >
+          Test-Drive
+        </Link>
```
Comment (SpaYco) [inline src/Pages/CarPage.js:22]: Clicking this button only redirects to the "my appointments" page, and doesn't use any fetching that would add this to the appointments, please fix that!

File: package.json [hunk 3]
```diff
@@ -6,15 +6,22 @@
...
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
     "redux": "^4.1.1",
+    "redux-logger": "^3.0.6",
     "redux-thunk": "^2.4.0",
     "reselect": "^4.1.0",
+    "universal-cookie": "^4.0.4",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
```
Comment (SpaYco) [inline package.json:27]: Please add tests to both the frontend and backend code

**Review by SpaYco — COMMENTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
Sorry for keeping you waiting all this time, we're using a new process for the final capstone and it can take a few days to get a review, but we're working on decreasing the time as soon as possible!

## Suggested changes

I checked the backend, so far, only testing would be needed there, however there are some changes needed in the front end!

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by nidalaa** [comment]
Hello @pountzas  and @carloso0114,

We would like to really apologize for the delayed code review. As you will see this review just before your evaluation meeting - please be careful with any changes. It might be better to leave your app in a working state for the presentation than to make last minutes changes :)

However, if you decide to make last-minute changes and because of that you will introduce some bugs - we will take that into account during your evaluation.

Good luck during the meeting!

Emily, Technical Curriculum Designer

**Pull request merged**